frontend/src/components/AnnotationRenderer.vue

```diff
@@ -154,8 +171,17 @@ export default {
     generateValidationTracker(config) {
       if (config) {
         this.validation = {}
+        this.condition = {}
         for (let elemConfig of config) {
           this.validation[elemConfig.name] = null
+          if (elemConfig["if"]) {
+            try {
+              this.condition[elemConfig.name] = jexl.compile(elemConfig["if"])
+            } catch (_) {
+              // error compiling expression -> treat the same as if the element
+              // did not have an "if" at all, and always show it.
+            }
+          }
```

Comment:
The point of doing this in here is so we just _compile_ the expressions once when the page loads, and can later _evaluate_ them repeatedly against different contexts.